cmake/options.cmake

```diff
@@ -3,7 +3,8 @@
 
 option(EBPFFORWINDOWS_ENABLE_TESTS "Set to true to enable tests" true)
 option(EBPFFORWINDOWS_ENABLE_INSTALL "Set to true to enable the install target")
-option(EBPFFORWINDOWS_ENABLE_DISABLE_EBPF_INTERPRETER "Set to true to compile with the interpreter always disabled")
+option(EBPFFORWINDOWS_DISABLE_JIT "Set to true to compile with the JIT compiler disabled")
+option(EBPFFORWINDOWS_DISABLE_INTERPRETER "Set to true to compile with the Interpreter disabled")
```

Comment:
Since the interpreter should be off by default, can we reverse the parity and require setting it to true to compile with the interpreter enabled?

Reply:
@dthaler This was discussed internally, would alter the UX on Visual Studio, where you will have to build via CLI if you want the Interpreter enabled.

Reply:
PS: the interpreter is disabled by default in `Release` builds